lightning.profile

```diff
@@ -23,6 +23,11 @@ function lightning_install_tasks() {
   $tasks['lightning_set_logo'] = [];
   $tasks['lightning_alter_frontpage_view'] = [];
 
+  $tasks['Drupal\lightning\Form\TelemetryOptInForm'] = [
+    'display_name' => t('Telemetry Opt-in'),
```

Comment:
We might wanna bikeshed the wording of this a bit.

Reply:
Do we need to run the language by Legal?

I propose something like the "Allow Lightning to send anonymous usage data to Acquia".

Reply:
I believe @grasmash is doing this:

> do not merge yet. Need to finalized the opt-in language first.

So that string might change.

Reply:
In case the string happens to stay the same, UI standards stipulate sentence case for headings:
```diff
-    $form['#title'] = "Telemetry Opt-in";
+    $form['#title'] = "Telemetry opt-in";
```